src/common/types/cpuArchitecture.ts

```diff
@@ -14,12 +18,19 @@ export enum OcmCpuArchitecture {
 
 export type SupportedCpuArchitecture = Extract<
   CpuArchitecture,
-  CpuArchitecture.x86 | CpuArchitecture.ARM
+  CpuArchitecture.x86 | CpuArchitecture.ARM | CpuArchitecture.ppc64le | CpuArchitecture.s390x
 >;
 
-export const getSupportedCpuArchitectures = (): SupportedCpuArchitecture[] => [
-  CpuArchitecture.x86,
-  CpuArchitecture.ARM,
-];
+export const getSupportedCpuArchitectures = (
+  versions?: OpenshiftVersionOptionType[],
+  selectedVersionValue?: string,
+): CpuArchitecture[] => {
```

Comment:
This needs to return `SupportedCpuArchitecture` to exclude the value `multi` which is valid for `CpuArchitecture` but not for `SupportedCpuArchitecture`